tracing-okhttp3/src/main/java/com/palantir/tracing/okhttp3/OkhttpTraceInterceptor.java

```diff
@@ -51,6 +51,9 @@ public Response intercept(Chain chain) throws IOException {
         if (span.getParentSpanId().isPresent()) {
             tracedRequest.header(TraceHttpHeaders.PARENT_SPAN_ID, span.getParentSpanId().get());
         }
+        if (span.getOriginatingSpanId().isPresent()) {
+            tracedRequest.header(TraceHttpHeaders.ORIGINATING_SPAN_ID, span.getOriginatingSpanId().get());
+        }
```

Comment:
Can we use the parent span ID instead of adding a new header?

Reply:
My reasoning was:

1. Extra header does not affect compliance with Zipkin API
2. But changing the semantics of what we're sending probably does.

Thoughts?

Reply:
Ah gotcha - I was thinking this was just local per-server. Makes sense to use a header if we want to push it all the way down.

Reply:
I don't believe this changes the semantics of what we're sending. In the unsampled case, we're still not suggesting data be sampled, but we're proving a more parent span ID which is optional for unsampled traces.
If there's something we think we want to enable that we're confident is in conflict with zipkin, I'm totally fine diverging, but if we can fit this into the spec that we've already bought into I think we're better off.

Reply:
I would like to change behaviour in the sampled as well as the unsampled case.

Reply:
or specifically: for sampled traces I would like access to the originating span id

Reply:
Makes sense, thanks for the clarification 👍